docs/application/tizen-studio/rt-ide/getting-started/overview.md

```diff
@@ -3,26 +3,32 @@
 The Tizen Studio for RT allows you to easily develop and test your RT application:
 
 1. [Install the Tizen Studio for RT](install.md) on your computer.
-2. [Manage the project](create.md), including creating a new Tizen Studio for RT project, creating application resources, and building the project.
+2. [Manage the project](create.md), which includes creating a new Tizen Studio for RT project, creating application resources, and building the project.
 3. [Flash the project](flash.md) on the board.
 4. [Use the serial terminal](terminal.md) to communicate with the board.
 5. [Debug the project](debug.md) to avoid runtime problems.
 
 
 ## Prerequisites
 
-Check the following prerequisites before installing the Tizen Studio for RT:
+To install the Tizen Studio for RT the following prerequistes must be checked:
 
-- Java Development Kit (JDK) requirements
+- You must install JDK and OpenJDK 8, 9 or 10 to use the Tizen Studio.
 
-  You must install a JDK 8 or 9 to use the Tizen Studio. Do not install OpenJDK.
+  To install the appropriate JDK and OpenJDK version for your Ubuntu system, go to [Ubuntu](https://www.ubuntu.com/) and follow the detailed instructions.
+ 
+- Operating system and hardware requirements:
 
+<<<<<<< HEAD
   To install the appropriate JDK version for your Ubuntu system, go to the Ubuntu Web site and follow the detailed instructions for installing the Oracle&reg; JDK version 8 or 9.
 
 
 - Operating system and hardware requirements
 
   The following table lists the supported operating systems and hardware requirements for the Tizen Studio for RT.
+=======
+  The following table lists the supported operating systems and hardware requirements for the Tizen Studio for RT:
+>>>>>>> Review of file
```

Comment:
Please review docs again after merge.

Reply:
@journey2w, Thank you for pointing out.
I will make the change.